test/expected/query-15.out

```diff
@@ -150,28 +150,54 @@ SHOW timescaledb.enable_optimizations;
 (5 rows)
 
 :PREFIX SELECT date_trunc('minute', time) t, avg(series_0), min(series_1), avg(series_2) FROM hyper_1_date GROUP BY t ORDER BY t DESC limit 2;
-                                                      QUERY PLAN                                                      
-----------------------------------------------------------------------------------------------------------------------
+                                                      QUERY PLAN                                                       
```

Comment:
This changed plan looks like a regression

Reply:
Same thing as I described below, should be fixed when we switch to using the cheapest full aggregation path as a basis for chunk-wise aggregation.